### PR TITLE
Fixes issue #31 - changed the api to tradier

### PR DESCRIPTION
**Pull request by Phantsure**
Used a new API(tradier) instead of previously used API as it was deprecated. The api needs access-token to be generated from the website [tradier.com](url). 
Bug:
-Delayed by 15mins

### CODE REVIEW

**Comment by ghoshbishakh** [comment]
Wow! Thanks for the effort! I wish I was free enough to test it myself.

Someone test it please.. :D

File: core/secret.py.example [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# create access token from tradier.com
+API_Access_Token="<Access-Token>"
```
Comment (agrawalo) [inline core/secret.py.example]: How do you obtain API access token?

**Review by agrawalo — COMMENTED** [review]
I generated my API token and restarted the app but it didnt work.

Can you please test this ?

File: core/secret.py.example (hunk 1 above)
Comment (Phantsure) [reply core/secret.py.example]: i am working on it. changing the api again

Comment (agrawalo) [reply core/secret.py.example]: Cool, which one are you planning to use now?

Comment (Phantsure) [reply core/secret.py.example]: the previous one was not deprecated, it was transferred. Using it again

**Pull request closed without merging**